pages/index.js

```diff
@@ -1,3 +1,172 @@
-export default function HomePage() {
-  return <h1>Hello from Next.js</h1>;
+import Image from "next/image";
+import Navbar from "./navbar";
+
+const users = [
```

Comment:
You might want to move your data `users` and `graffitiPosts` into separate files in the future 
and import them here, to keep the `HomePage` code clean.